ApplyToBecomeInternal/ApplyToBecomeInternal/Services/DocumentGeneration/HtmlDocumentFactory.cs

```diff
@@ -0,0 +1,75 @@
+﻿using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using AngleSharp.Html.Parser;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace ApplyToBecomeInternal.Services.WordDocument
+{
+	public static class HtmlDocumentFactory
+	{
+		private static string[] _tags = new[] { "ol", "ul", "p" };
+
+		public static IHtmlDocument Create(string html)
+		{
+			var htmlParser = new HtmlParser();
+			var sanitisedHtml = SanitiseHtml(html);
+			var document = htmlParser.ParseDocument(sanitisedHtml);
+			var groupedElements = GroupElements(document);
+			foreach (INode child in groupedElements.SelectMany(l => l))
+			{
+				child.RemoveFromParent();
+			}
+			ReconstructDocumentBody(document, groupedElements);
+			return document;
+		}
+
+		private static string SanitiseHtml(string html)
+		{
+			return Regex.Replace(html, @"<\/?(br|div|span)>", "");
```

Comment:
For what it's worth - `br` is a valid tag for user input. If a user wanted a new line inside a bullet point you can do `shift+enter` and it inserts a `br` tag for the new line

Reply:
(You can also do this with paragraphs, but i'm not sure why someone would. But `shift+enter` works the same there)